lib/Service/CardService.php

```diff
@@ -51,6 +51,10 @@ public function __construct(CardMapper $cardMapper, StackMapper $stackMapper, Pe
 		$this->attachmentService = $attachmentService;
 	}
 
+	public function fetchDeleted($boardId) {
+		return $this->cardMapper->findDeleted($boardId);
```

Comment:
We need a call to checkPermissions like in the other methods, so only users with proper permissions can access this.